index.html

```diff
@@ -180,7 +180,9 @@ <h1 class="title is-1 pb-6">
         <div class="container">
             <h1 class="title is-1"><a name="showcase">#</a> Showcase</h1>
             <br>
-            <p class="subtitle is-4">Made something cool with <b>wgpu</b>? <a href="https://github.com/gfx-rs/wgpu-rs.github.io">Make a PR</a>, and reach out to us on <a href="https://matrix.to/#/#wgpu-users:matrix.org">#wgpu-users</a>!</p>
+            <p class="subtitle is-4">Made something cool with <b>wgpu</b>? <a
```

Comment:
I realize my linter auto-formatted some stuff and I was working quickly. :) I can revert this but it might not be necessary!